### DISCUSSION TITLE
“Hold” looping playback for samples in kit

### DISCUSSION THREAD

**Discussion by Phonowav** [SynthstromAudible/DelugeFirmware#2906] [Ideas]
Currently, a sample in kit can be set to loop but the loop only triggers by holding down a pad. With the proposed “hold” option, a single pad press will trigger the sample loop. Loop can be cut off with another single pad press, or choked by another sample in the choke group.

This will be especially useful for an SP404-type workflow in drum kit view!

**Comment by RJ-Eckie** [comment]
I read about this on facebook the other day, sounds fun and useful! I would propose the following naming structure, from a perspective of aligning better with broadly used terms:
- momentary once (was **cut**)
- momentary loop (was **loop**)
- trigger once (was **once**)
- trigger loop (SP404 behavior)
- stretch (unchanged)

**Reply by Phonowav** [reply]
Awesome! A year later…think this feature could come to be? It would change the game! It seems like it would be a simple tweak since the kits already have so many different playback modes
